src/v/serde/parquet/encoding.cc

```diff
@@ -22,10 +22,11 @@ iobuf encode_plain(const chunked_vector<boolean_value>& vals) {
     while (i < vals.size()) {
         size_t pass_size = std::min<size_t>(vals.size() - i, CHAR_BIT);
         uint8_t tmp = 0;
-        for (; i < pass_size; ++i) {
-            tmp >>= 1U;
-            static constexpr uint8_t leading_bit = 0b1000'0000;
-            tmp |= vals[i].val ? leading_bit : 0U;
+        size_t shift = 0;
+        for (; shift < pass_size; ++i, ++shift) {
+            if (vals[i].val) {
+                tmp |= 1U << shift;
+            }
```

Comment:
so the previous commit wasn't plain encoding?

Reply:
Those commits should be squashed together. Let me do that (sorry I thought I did).